bazel/markdown_to_html/src/extensions/docs-code/docs-code-multifile.ts

```diff
@@ -57,12 +57,15 @@ export const docsCodeMultifileExtension = {
     return undefined;
   },
   renderer(this: RendererThis, token: DocsCodeMultifileToken) {
-    const attributes = buildAttr('path', token.path) + buildAttr('preview', token.preview);
-
-    return `
-    <div class="docs-code-multifile"${attributes}>
+    const el = JSDOM.fragment(`
+    <div class="docs-code-multifile">
     ${this.parser.parse(token.paneTokens)}
     </div>
-    `;
+    `).firstElementChild!;
+
+    token.path && el.setAttribute('path', token.path);
+    token.preview && el.setAttribute('preview', `${token.preview}`);
```

Comment:
I'm not really a fan of using `&&` as a shorthand `if` statement- can we make these actual conditionals? 

(here and elsewhere)